.github/workflows/node.js.yml

```diff
@@ -21,6 +21,7 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
           cache: "npm"
+      - run: corepack enable
```

Comment:
Just curious, why did you add this ?

Reply:
this is a required setup step for the latest yarn: https://yarnpkg.com/getting-started/install